common/persistence/src/main/resources/db/migration/V11__alterScanInstanceId.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE scan_instance ALTER COLUMN id TYPE bigint;
```

Comment:
What about the ALTER SEQUENCE?

Reply:
We tested this locally without alter sequence. We tried both scenarios, the one that we have in prod, where we run the script already, and the one we have in WRUs, INT where they are not run. But for safety, I think you are right, we better add it now.